src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -170,7 +170,7 @@ namespace AppInstaller::Filesystem
 
     bool VerifySymlink(const std::filesystem::path& symlink, const std::filesystem::path& target)
     {
-        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlink);
+        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlink).lexically_normal();

```

Comment:
Since here it's doing direct string comparison, it would be better to use weakly_canonical to normalize the case of the path.

Also the above one(during create_symlink) can also use  weakly_canonical to get case correct.